File: 3190.find-minimum-operations-to-make-all-elements-divisible-by-three.java
/*
 * @lc app=leetcode id=3190 lang=java
 *
 * [3190] Find Minimum Operations to Make All Elements Divisible by Three
 */

// @lc code=start
class Solution {
    public int minimumOperations(int[] nums) {
        int count = 0;
        for (int i = 0; i<nums.length;i++) {
            if (nums[i]%3 != 0) {
                count++;
            }
        }
        return count; 
    }
}
// @lc code=end

File: 3190.find-minimum-operations-to-make-all-elements-divisible-by-three.py
#
# @lc app=leetcode id=3190 lang=python
#
# [3190] Find Minimum Operations to Make All Elements Divisible by Three
#

# @lc code=start
class Solution(object):
    def minimumOperations(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        count = 0
        for i in nums:
            if i%3 != 0:
                count += 1
        return count
         
# @lc code=end

